pages/docs/reference/python/migrations/v0.3-to-v0.4.mdx

```diff
@@ -0,0 +1,57 @@
+# v0.3 to v0.4 migration guide
```

Comment:
```suggestion
# Migration guide for v0.3 to v0.4 Python SDK 
```

I'm on the fence here.

It makes sense to keep the title as you suggested because users will infer the context from the sidebar. That being said, the SEO/algolia will not.

Do you think this version is too verbose?